app/controllers/DeploymentApprovalController.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import config.AppConfig
+import connectors.AuthConnector
+import javax.inject.Inject
+import model._
+import play.api.Play.current
+import play.api.data.Form
+import play.api.i18n.Messages.Implicits._
+import play.api.mvc.{Action, AnyContent}
+import services.DeploymentApprovalService
+import uk.gov.hmrc.http.HeaderCarrier
+import utils.GatekeeperAuthWrapper
+import views.html.deploymentApproval.{deploymentApproval, deploymentReview}
+
+import scala.concurrent.Future
+
+class DeploymentApprovalController @Inject()(val authConnector: AuthConnector,
+                                             deploymentApprovalService: DeploymentApprovalService)(override implicit val appConfig: AppConfig)
+  extends BaseController with GatekeeperAuthWrapper {
+
+  def pendingPage(): Action[AnyContent] = requiresAtLeast(GatekeeperRole.USER) { implicit request => implicit hc =>
```

Comment:
On all of these methods, drop the implicits down to the next line for readability and to avoid scrolling  in GitHub.